Automattic/wp-calypso
# Move the pagination component to shared `/components` folder

A few things in Store are going to use a pagination component, and the Products list is already grabbing this one from `my-sites/stats`. It needs some updates both in code style and to enable keyboard accessibility, and I've also moved it to the shared `components` directory.

Each element was able to take keyboard focus, but with no keydown handler you couldn't actually click the element. The fix was to use a borderless button for each clickable element - `button`s automatically handle keyboard events as click events. This also fixes a bug where inactive arrows could be focused and the focus style was applied.

![screen shot 2017-07-17 at 6 43 16 pm](https://user-images.githubusercontent.com/541093/28292997-d18926f6-6b1f-11e7-8517-e948e6902f1c.png)

To test:

- Check out the new component in devdocs: `/devdocs/design/pagination`
- Check that existing uses of the component still look/work correctly. It's is being used on 
  - `/stats/follows/comment/:site`
  - `/store/products/:site`
  - `/comments/:site` (theoretically, I couldn't get it to display there -- total was always 0)
